packages/headless/src/app/search-engine/search-engine-configuration-options.ts

```diff
@@ -0,0 +1,92 @@
+import {RecordValue, Schema, StringValue} from '@coveo/bueno';
+import {PreprocessRequestMiddleware} from '../../api/platform-client';
+import {
+  PostprocessFacetSearchResponseMiddleware,
+  PostprocessQuerySuggestResponseMiddleware,
+  PostprocessSearchResponseMiddleware,
+} from '../../api/search/search-api-client-middleware';
+import {localeValidation} from '../../features/configuration/configuration-actions';
+import {
+  engineConfigurationOptionDefinitions,
+  EngineConfigurationOptions,
+} from '../engine-configuration-options';
+
+export interface SearchEngineConfiguration extends EngineConfigurationOptions {
```

Comment:
It would be more consistent to keep interfaces name similar:
`SearchEngineConfigurationOptions extends EngineConfigurationOptions`
or
`SearchEngineConfiguration extends EngineConfiguration`